integration_test/start.test.ts

```diff
@@ -0,0 +1,212 @@
+import { Account } from 'web3/eth/accounts';
+import { FacilitatorConfig } from '../src/Config/Config';
+
+import Utils from './Utils';
+import MosaicConfig from '../src/Config/MosaicConfig';
+import StakeRequest from '../src/models/StakeRequest';
+
+const Web3 = require('web3');
+
+import Timeout = NodeJS.Timeout;
+
+const facilitatorInit = path.join(__dirname, 'facilitator_init.sh');
+const facilitatorStart = path.join(__dirname, 'facilitator_start.sh');
+const facilitatorKill = path.join(__dirname, 'kill_facilitator_process.sh');
+
+const originWeb3 = new Web3('http://127.0.0.1:41515');
+const auxiliaryWeb3 = new Web3('http://127.0.0.1:41000');
+originWeb3.transactionConfirmationBlocks = 6;
+auxiliaryWeb3.transactionConfirmationBlocks = 6;
+const exportWorkerPrefix = 'MOSAIC_ADDRESS_PASSW_';
+
+describe('facilitator start', () => {
+  const mosaicConfig = MosaicConfig.fromFile(path.join(__dirname, 'mosaic.json'));
+
+  const ostComposer: string | undefined = mosaicConfig.originChain.contractAddresses.ostComposerAddress;
+
+  const auxChainId = 1000;
+  const outputOptions = [process.stdout, process.stderr];
+
+  let facilitatorConfig: FacilitatorConfig;
+  let originWorker: string;
+  let utils: Utils;
+  let simpleTokenInstance: EIP20Token;
+  let stakerAccount: Account;
+  let originFunder: string;
+  let auxiliaryFunder: string;
+  let auxWorkerExport;
+  let originAnchorInterval: Timeout;
+  const stakerOSTBalance = 20000;
+  const stakeRequest = new StakeRequest(
+    '',
+    new BigNumber(10),
+    '',
+    new BigNumber(0),
+    new BigNumber(0),
+    new BigNumber(1),
+    '',
+  );
+
+  const reward = stakeRequest.gasPrice!.mul(stakeRequest.gasLimit!);
+  const mintedAmount = stakeRequest.amount!.sub(reward);
+
+  before(async () => {
+    const originAccounts = await originWeb3.eth.getAccounts();
+    const auxiliaryAccounts = await auxiliaryWeb3.eth.getAccounts();
+    originFunder = originAccounts[4];
+    auxiliaryFunder = auxiliaryAccounts[6];
+    utils = new Utils(originWeb3, auxiliaryWeb3, originFunder, auxiliaryFunder, mosaicConfig);
+
+    // Initializing facilitator config.
+    execSync(facilitatorInit, { stdio: outputOptions });
+  });
+
+  it('fund staker', async () => {
+    simpleTokenInstance = utils.getSimpleTokenInstance();
+    stakerAccount = originWeb3.eth.accounts.create('simpletoken');
+    originWeb3.eth.accounts.wallet.add(stakerAccount);
+    stakeRequest.beneficiary = stakerAccount.address;
+
+    const transferRawTx: TransactionObject<boolean> = await simpleTokenInstance.methods.transfer(
+      stakerAccount.address,
+      stakerOSTBalance,
+    );
+
+    const transferReceipt = await utils.sendTransaction(
+      transferRawTx,
+      {
+        from: originFunder,
+        gasPrice: await originWeb3.eth.getGasPrice(),
+      },
+    );
+    await utils.verifyERC20Transfer(transferReceipt, stakerAccount.address, stakerOSTBalance);
+
+    await utils.fundETHOnOrigin(stakerAccount.address);
```

Comment:
If eth transfer fails, transaction would fail.
We could have a method(here, transfer method) which would not do any operation and would still succeed. Hence, i did verification of ERC20 transfer.